fern/docs.yml

```diff
@@ -6,64 +6,167 @@ instances:
+            path: ./overview/dummy.mdx
+          - page: Integration Tests
+            path: ./overview/dummy.mdx
+          - page: Code Snippets
+            path: ./overview/dummy.mdx
+          - page: Augment with custom code
+            path: ./overview/dummy.mdx
+          - page: Merging multiple APIs
+            path: ./overview/dummy.mdx
+          - page: GitHub integration
+            path: ./overview/dummy.mdx
+  - section: Docs
+    contents: 
+      - page: Overview
+        path: ./overview/dummy.mdx
+      - page: Quickstart
+        path: ./overview/dummy.mdx
+      - section: Global Settings
+        contents: 
+          - page: URLs
+            path: ./overview/dummy.mdx
+          - page: Colors
+            path: ./overview/dummy.mdx
+          - page: Layout
+            path: ./overview/dummy.mdx
+          - page: Navigation
+            path: ./overview/dummy.mdx
+          - page: Typography
+            path: ./overview/dummy.mdx
+          - page: CSS and JS Overrides
+            path: ./overview/dummy.mdx
+          - page: Call to actions
+            path: ./overview/dummy.mdx
+      - section: Writing Content
+        contents: 
+          - page: Page Titles and Metadata
+            path: ./overview/dummy.mdx
+          - page: Headers and Text
+            path: ./overview/dummy.mdx
+          - page: Images, Videos and Embeds
+            path: ./overview/dummy.mdx
+          - page: Lists and Tables
+            path: ./overview/dummy.mdx
+          - page: Code Blocks
+            path: ./overview/dummy.mdx
+      - section: API References
+        contents: 
+          - page: API Definition Setup
+            path: ./overview/dummy.mdx
+          - page: Errors, Audiences, and Snippets 
+            path: ./overview/dummy.mdx
+          - page: API Playground
+            path: ./overview/dummy.mdx
+      - section: Custom Components
+        contents: 
+          - page: Cards
+            path: ./overview/dummy.mdx
+          - page: CodeBlocks
+            path: ./overview/dummy.mdx
+          - page: Callouts
+            path: ./overview/dummy.mdx
+          - page: Icons
+            path: ./overview/dummy.mdx
+          - page: Availability
+            path: ./overview/dummy.mdx
+          - page: Tabs
+            path: ./overview/dummy.mdx
+          - page: Accordion
+            path: ./overview/dummy.mdx
+          - page: Steps
+            path: ./overview/dummy.mdx
+          - page: Endpoint Snippets
+            path: ./overview/dummy.mdx
+  - section: Configuration
+    contents: 
+      - section: Fern Folder
         contents:
-          # TBD
-          # - page: Overview
-          #   path: ./docs/pages/overview/define-your-api/api-overview.mdx
-          - section: OpenAPI
-            slug: openapi
-            contents:
+          - page: Overview
+            path: ./overview/dummy.mdx
+          - page: generators.yml
+            path: ./overview/dummy.mdx
```

Comment:
When we get support asks about `generators.yml` we should be able to link here. There is no high-level explanation of the `fern` folder and `generators.yml` in our docs today.